activemq-jaas/src/main/java/org/apache/activemq/jaas/LDAPLoginModule.java

```diff
@@ -175,7 +171,7 @@ public boolean commit() throws LoginException {
     public boolean abort() throws LoginException {
         if (!succeeded) {
             return false;
-        } else if (succeeded && commitSucceeded) {
```

Comment:
Why is the logic changing here?

Reply:
succeeded is alway `true` in line 174. Logic hasn't been changed

Reply:
I agree with @Dm-Chebotarskyi : as we already test `succeeded` at line 172, no need to test again.